.editorconfig

```diff
@@ -0,0 +1,7 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Done

> On Jan 11 2016, at 7:46 pm, Ari Porad &lt;notifications@github.com&gt;
wrote:  

>

> In [.editorconfig](https://github.com/shelljs/shelljs/pull/249#discussion_r4
9406606):

>  
>  
>     > @@ -0,0 +1,7 @@

>     > +# EditorConfig is awesome: http://EditorConfig.org

>  
>

> Thanks!

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/shelljs
/shelljs/pull/249/files#r49406606).![](https://github.com/notifications/beacon
/AC4eoCEa364dL2EEC7J07Qx7hdq2tyMAks5pZFJ2gaJpZM4GcyGK.gif)